pythia6.sh

```diff
@@ -1,59 +1,35 @@
+# a pythia6 recipe based on the one from FairROOT
 package: pythia6
-version: "426"
+version: "%(tag_basename)s%(defaults_upper)s"
+tag: "416"
+source: https://github.com/alisw/pythia6.git
 ---
 #!/bin/sh
-curl -O -fSsL --insecure http://cern.ch/service-spi/external/MCGenerators/distribution/pythia6/pythia6-426-src.tgz
-tar xzvf pythia6-426-src.tgz
-cd  pythia6/426
 
-case $ARCHITECTURE in
-  slc5*) 
-    PLATF_CONF_OPTS="--enable-shared"
-    PLATF_LDFLAGS=""
-    PLATF_LD=""
-    F77="`which gfortran`"
-  ;;
-  osx*) 
-    PLATF_CONF_OPTS="--disable-shared --enable-static"
-    PLATF_LDFLAGS=""
-    PLATF_LD="LD='`which gcc`'" 
-    F77="`which gfortran` -fPIC"
-  ;;
-  *) 
-    PLATF_CONF_OPTS="--disable-shared --enable-static"
-    PLATF_LDFLAGS=""
-    PLATF_LD=""
-    F77="`which gfortran` -fPIC"
-  ;;
-esac
+cmake ${SOURCEDIR} \
```

Comment:
Bonus points if you align the "\" ...;) `:'<,'>Tabularize / \` on vim + tabularize plugin.